### PR TITLE
#160283217 Update readme

### PR DESCRIPTION
**Pull request by tutugodfrey**
#### What does this PR do?
- Update readme

#### Description of Task to be completed?
- Add description of the application
- outline how the application can be started
- add the link to API Docs for building the application
- Add a table of the path/ routes available

#### How should this be manually tested?
- N/A

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
#160283217 

#### Screenshots (if appropriate)
-  N/A

#### Questions:
- N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+- Users can follow their favorite authors and subscribe for notifications when authors post new articles 
+- Authors get notifications when their articles are read, liked, favorited or when users follow them. To access all the feature of the application, a user will have to create an acount.
+
+## Documentation
+Documentation of the API for this project is hosted on https://authorhaven.herokuapp.com/api-docs
+
+## Starting the application in development
+Clone the application to your local machine. In your terminal run `npm install` to install dependencies. Then run `npm run dev` to start the application. The application will start and open in your default browser.
+
+## Relevant urls
```
Comment (rhenshaw56) [inline README.md]: you don't need this api methods in here, take them off

**Review by rhenshaw56 — CHANGES_REQUESTED** [review]
The usual template for writing a readme usually has  the following headers.
FEATURES|GETTING STARTED|TECHNOLOGIES|CONTRIBUTIONS

I see none here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
-# rivendell-ah-client
+# rivendell-ah-client Author's Haven
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging)
+
+Authors Haven is am online application that bring together a community of authors and readers. It allows Authors to create and post articles based on topic of interest or popular demands. Reader can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes:
```
Comment (JCanaks) [inline README.md]: correct typos  on  description it should be `Author's haven is 'an' online application that 'brings' together a community of authors .....It allows 'authors' (lowercase a)...... based on 'their' .....or popular 'demand' (singular). 'Readers' (plural)`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
-# rivendell-ah-client
+# rivendell-ah-client Author's Haven
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging)
+
+Authors Haven is am online application that bring together a community of authors and readers. It allows Authors to create and post articles based on topic of interest or popular demands. Reader can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
+- Users are able to give ratings for articles
+- Users can like /dislike articles
+- Users are able to mark articles as favorite
```
Comment (JCanaks) [inline README.md]: simplify to  `users can mark an article as a favorite`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,54 @@
-# rivendell-ah-client
+# rivendell-ah-client Author's Haven
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging)
+
+Authors Haven is am online application that bring together a community of authors and readers. It allows Authors to create and post articles based on topic of interest or popular demands. Reader can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
+- Users are able to give ratings for articles
```
Comment (JCanaks) [inline README.md]: simplify to  `Users can rate an article`

**Review by rhenshaw56 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,61 @@
 # rivendell-ah-client
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging) [![Test Coverage](https://api.codeclimate.com/v1/badges/752141b5976d9ebab531/test_coverage)](https://codeclimate.com/github/andela/rivendell-ah-client/test_coverage)
+
+Authors Haven is online application that brings together a community of authors and readers. It allows authors to create and post articles based on their topic of interest or popular demand. Readers can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes:
```
Comment (JCanaks) [inline README.md]: Author's haven is 'an' online

File: README.md [hunk 6]
```diff
@@ -1,2 +1,61 @@
...
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging) [![Test Coverage](https://api.codeclimate.com/v1/badges/752141b5976d9ebab531/test_coverage)](https://codeclimate.com/github/andela/rivendell-ah-client/test_coverage)
+
+Authors Haven is online application that brings together a community of authors and readers. It allows authors to create and post articles based on their topic of interest or popular demand. Readers can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
+To access all the feature of the application, a user will have to create an acount.
+
+## Features
+- Users can register with Oauth services (facebook, linkedin, twitter)
+- Users can rate articles
+- Users can like /dislike articles
+- Users Can mark articles as favorite
```
Comment (JCanaks) [inline README.md]: as 'a' favorite

**Review by JCanaks — APPROVED** [review]

**Review by mppdg — APPROVED** [review]

**Pull request merged**